runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx

```diff
@@ -19,22 +24,22 @@ interface Params extends Record<string, string> {
     slug: string;
 }
 
-export const getServerSideProps: GetServerSideProps<Props, Params> = async ({ res, params }) => {
+export const getServerSideProps: GetServerSideProps<Props, Params> = async ({ res, params, locale }) => {
     let post: BlogPost | undefined;
 
-    if (isCacheUpToDate()) {
-        post = getSinglePostCache(params.slug);
+    if (isCacheUpToDate(locale as Language)) {
```

Comment:
It shouldn't be null because locale is embedded into URL like this: /pl/blog
So, if it's undefined or null it's not valid route.

But I added check.